### DISCUSSION TITLE
MS/Office 365 Support

### DISCUSSION THREAD

**Discussion by Geschan** [outline/outline#2796] [Ideas]
It would be great if Outline would support Microsoft 365 (aka Office 365).
In particular support for:

- Word/Powerpoint/Excel Online (like the Google Docs support)
- Onedrive for Business/Sharepoint
- Microsoft Planner (like Trello)
- Microsoft Teams (like Slack)

This would help to make Outline super usefull for companies that are using MS365.

**Comment by tommoor** [comment]
Could you describe what you mean by support for these apps?

**Reply by kmanojkumar** [reply]
Hi @tommoor,

The MS365 tools can just have similar capabilities in Outline as their Google counterparts. 

Like, Google Docs paste a link feature can be used for MS Word, PowerPoint and Excel documents. 

Teams can have similar capabilities as Slack with Search Outline Docs and Bot Notifications #2093.

**Reply by Geschan** [reply]
Hi @tommoor
as @kmanojkumar mentioned, the integration should have similar capabilities in Outline as their counterparts mentioned in my request.
So for example pasting files from Onedrive/Sharepoint with same behavior as Google Drive. Pasting Excel, Word, Powerpoint documents from OneDrive/Dropbox/... with access to the Office webapps.

**Reply by HectorB-2020** [reply]
> Could you describe what you mean by support for these apps?

It woould be nice to be able to import Excel files for example.
I can now import Word (**.docx**) but not Excel (**.xlsx**).
Going further into OSS, I'd vote for importing LibreOffice Calc (**.ods**) and LibreOffice Writer (**.odt**).

**Reply by HectorB-2020** [reply]
One more similar request: https://github.com/outline/outline/discussions/3542#discussion-4077277

**Comment by ejqs** [comment]
+1 for Teams integration
